operators/trim_left_or_right_handles.py

```diff
@@ -65,3 +75,19 @@ def select_strip_handle(self, sequences, side, frame):
                 s.select = False
             if handle_side:
                 bpy.ops.sequencer.select_handles(side=handle_side)
+
+    def ripple_sequences(self, sequence, context):
+        gap = 0
+        side = 'RIGHT' if sequence.select_right_handle else 'LEFT'
+        if side == 'RIGHT':
+            gap = sequence.frame_final_start + sequence.frame_duration - sequence.frame_final_end
```

Comment:
This doesn't work at all: this gives a negative value. You can't calculate the gap this way, the gap is the original frame_final_end minus the final one. Anyway, the tool doesn't work in blender, it moves rippled strips before the 1st frame in the VSE here.